deploy/kubernetes/base/node.yaml

```diff
@@ -18,11 +18,9 @@ spec:
           image: quay.io/k8scsi/driver-registrar:v0.4.1
           args:
             - "--v=5"
-            - "--csi-address=$(ADDRESS)"
+            - "--csi-address=/var/lib/kubelet/plugins/com.google.csi.gcepd/csi.sock"
```

Comment:
I know this is unrelated but since you're touching these files can you also change the `serviceAccount` field in this yaml to the non-deprecated `serviceAccountName` :) :) :)

Reply:
done